README.md

```diff
@@ -4,19 +4,26 @@ It is year 2 for our [jaffle shop](https://github.com/dbt-labs/jaffle_shop) and
 
 We are tasked to understand what kind of jaffles we make the most money from.
 
-So we decided to run a [clustering algorithm](https://github.com/fal-ai/jaffle_shop_with_fal/blob/main/clustering.py) to separate the orders into 3 different clusters and then to calculate all the [revenue for each cluster](https://github.com/fal-ai/jaffle_shop_with_fal/blob/main/models/cluster_stats.sql).
+So we decided to run a [clustering algorithm](https://github.com/fal-ai/jaffle_shop_with_fal/blob/main/clustering.py) to separate the orders into 3 different clusters and then to calculate all the [revenue for each cluster](https://github.com/fal-ai/jaffle_shop_with_fal/blob/main/models/cluster_stats.sql). Once our clustering algorithm is done, we want to send a notification to a Slack channel notifying our team that the results are ready for viewing.
 
-[Fal](https://github.com/fal-ai/fal) is the perfect tool for the task at hand, by using the [`fal flow`](https://blog.fal.ai/python-or-sql-why-not-both/) command, we can run our clustering python script in the middle of 2 dbt models.
+The [fal](https://github.com/fal-ai/fal) + [dbt-fal](https://github.com/fal-ai/fal/tree/main/adapter) combination is the perfect tool for the task at hand:
+
+- Using the `dbt-fal` adapter, we can iterate on and ship our clustering algorithm written in Python with any datawarehouse **right within our dbt project.**
+
+- Using the `fal flow` CLI command, we can send a Slack notification to notify our team.
+
+With this combo, you won't to leave your dbt project while still bringing more capabilities to your stack.
 
 ### Installing Instructions:
 
-1. Install fal
+1. Install `fal` and `dbt-fal`
 
 ```
-$ pip install fal
+$ pip install fal dbt-fal[duckdb, postgres, redshift, bigquery]
```

Comment:
This seems a bit ambiguous to me. Maybe we can select one adapter and let the users know that they can install others.

Reply:
Sounds good! I will switch it to duckdb :)

Reply:
👍